### ISSUE TITLE
`PartialInstanceVariable` reports false positive for templates under folders with underscores

### ISSUE THREAD

**Issue by joemasilotti** [Shopify/erb-lint#244]
I have a template `app/views/cold_messages/new.html.erb` that uses an instance variable and `PartialInstanceVariable` is reporting a violation.

Open source app and failing CI run here: https://github.com/joemasilotti/railsdevs.com/runs/4451151656?check_suite_focus=true

```
Run bundle exec erblint --lint-all
1 error(s) were found in ERB files
Linting 55 files with 16 linters...

Instance variable detected in partial.
In file: app/views/cold_messages/new.html.erb:1

Error: Process completed with exit code 1.
```

```
# app/views/cold_messages/new.html.erb
<%= render "shared/error_messages", resource: @message %>

<%# More code follows... %>
```

Digging into the code, it looks like the regex naively checks for an underscore to identify a partial. Is this correct, or am I misunderstanding the validator?

```ruby
return unless processed_source.filename.match?(%r{(\A|.*/)_[^/\s]*\.html\.erb\z}) &&
```

**Comment by joemasilotti** [comment]
Ah, I see that this was fixed in #235!

Any chance a release can be cut to include this bug fix?
